### PR TITLE
add sdn to ovn migration workflow and cluster profile for IPI AWS

### PR DESCRIPTION
**Pull request by weliang1**
add sdn to ovn migration workflow and cluster profile for IPI AWS for https://issues.redhat.com/browse/OCPQE-9875

@zhaozhanqi @anuragthehatter @lihongan PTAL!

### CODE REVIEW

**Comment by zhaozhanqi** [comment]
@weliang1 you can follow the failed job logs to fix related issue.

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/sdn2ovn/provision/cucushift-installer-rehearse-aws-ipi-ovn-sdn2ovn-provision-chain.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-ovn-sdn2ovn-provision
+  steps:
+  - chain: ipi-conf-aws
+  - chain: ipi-install
+  - ref: ovn-sdn-migration
+  - ref: ovn-conf
```
Comment (zhaozhanqi) [inline ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/sdn2ovn/provision/cucushift-installer-rehearse-aws-ipi-ovn-sdn2ovn-provision-chain.yaml:7]: do we still need  `ref: ovn-conf` after sdn migration to ovn

Comment (zhaozhanqi) [reply ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/sdn2ovn/provision/cucushift-installer-rehearse-aws-ipi-ovn-sdn2ovn-provision-chain.yaml]: it's ok under OVN

**Comment by jianlinliu** [comment]
pls squash commits into one.

**Pull request merged**